Sources/App/Models/Package.swift

```diff
@@ -271,6 +271,10 @@ extension QueryBuilder where Model == Package {
     func filter(by url: URL) -> Self {
         filter(\.$url == url.absoluteString)
     }
+
+    func filter(by urls: some Collection<URL>) -> Self {
+        filter(\.$url ~~ urls.map(\.absoluteString))
```

Comment:
There is a `SQLKit` based way to make this a case-insensitive filter but it's a bit more complicated. I'll leave that to a follow-up.

Reply:
When doing that, we should also canonicalise the urls from the custom collection. Since they are not added via our normal package list mechanism and validated (canonicalised) in the process, we can't assume they're all as well-formed as they are in the main package list.